README.md

```diff
@@ -53,6 +54,14 @@ $ npm run docs
 
 ### Pushing to webeditor
 
+You need to first login into [clasp](https://github.com/google/clasp#login):
```

Comment:
Wouldn't it be better if it was
You first need to log into ...
Otherwise, it looks good to me

Reply:
Fixed